docs/markdown/snippets/project_options_in_machine_files.md

```diff
@@ -0,0 +1,51 @@
+## Project and Meson options can be set in native or cross files
+
+A new set of sections has been added to the cross and native files, `[project
+options]` and `[<subproject_name>:project options]`, where `subproject_name`
+is the name of a subproject. Any options that are allowed in the project can
+be set from this section. They have the lowest precedent, and will be
+overwritten by command line arguments
+
+
+```meson
+option('foo', type : 'string', value : 'foo')
+```
+
+```ini
+[project options]
+foo = 'other val'
+```
+
+```console
+meson build --native-file my.ini
+```
+
+Will result in the option foo having the value `other val`,
+
+```console
+meson build --native-file my.ini -Dfoo='different val'
+```
+
+Will result in the option foo having the value `different val`,
+
+
+for subprojects this looks like:
+
+```ini
+[zlib:project options]
+foo = 'some val'
+```
+
+Additionally meson level options can be set in the same way, using the
+`[built-in options]` section.
+
+```ini
+[built-in options]
+c_std = 'c99'
+```
+
+Subprojects works too, although only `default_library` can currently be set:
```

Comment:
```suggestion
Subprojects works too, although only `default_library` and `werror` can currently be set:
```
Because https://github.com/mesonbuild/meson/pull/6795

Reply:
I swear I added `werror` already, I must have accidentally rebased it out.

Reply:
It *was* added, but not here (rather in the other part of the doc) =)

Reply:
Removed extra `s` as well.
```suggestion
Subprojects work too, although only `default_library` and `werror` can currently be set:
```